CONTRIBUTORS_GUIDE.md

```diff
@@ -0,0 +1,108 @@
+# 🌟 Contributing Guide | PayStell
+
+We’re excited you’re interested in contributing! Please follow these steps to ensure a smooth and efficient collaboration process.
+
+🎉 Thank you for being interested in contributing to the PayStell project! 🎉
+
+Feel welcome and read the following sections in order to know how to ask questions and how to work on something.
+
+Please make sure you are welcoming and friendly in all of our spaces.
+
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition. 👏
+
+## 👨🏻‍💻 Issues
+
+The best way to contribute to our projects is to apply to the issues <a href="https://github.com/PayStell/paystell-website/issues" target="_blank">here</a> .
+
+
+---
+
+## 1️⃣ Fork the Repository
+
+- Click the **Fork** button in the top-right corner to create a copy of the repository under your account.
+
+---
+
+## 2️⃣ Clone the Fork
+
+- Clone the forked repository to your local machine by running the following command:
+
+   ```bash
+   git clone https://github.com/YOUR_USERNAME/REPOSITORY_NAME.git
+   ```
+
+- Replace `YOUR_USERNAME` and `REPOSITORY_NAME` with your GitHub username and the repository name.
+
+---
+
+## 3️⃣ Create a new branch or use the main branch
+
+- Create a branch name based on the type of change (e.g., `feat/name-related-issue`, `docs/name-related-issue`).
+
+   ```bash
+   git checkout -b branch-name
+   ```
+
+   > Example: `docs/update-readme` or `fix/bottom-bug`.
+
+---
+
+## 4️⃣ Make Changes and Write Atomic Commits
+
+- Make changes in your local repository
+- Follow **atomic commit principles**:
+
+   - Each commit should address a single, logical change.
+   - Avoid bundling unrelated changes in a single commit. In case you want to add additional items, please ask the mainteiners first.
+   - Write clear and descriptive commit messages using the format:
+
+   ```bash
+   git add .
+   git commit -m "[type] description"
```

Comment:
Remove [ ]. Use prefixes like feat: or fix: for all types.

Reply:
Okay, I'm going to work on that.